src/NodeApi/JSDispatcherQueue.cs

```diff
@@ -0,0 +1,221 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Hermes.Example;
+
+public delegate void JSTypedEventHandler<TSender, TResult>(TSender sender, TResult args);
```

Comment:
I'm not opposed to typed event-handler delegates in general, but is there any particular benefit of having it here, over normal `EventHandler<TArgs>`?

Reply:
Changed. I copied it from the WinRT `DispatcherQueue`, but does not make much sense in this context.